architecturalOverview.md

```diff
@@ -1,2 +1,7 @@
 # Project Overview
-The purpose of the project is to provide clean and minimalistic implementations of basic algorithms and data structures in Python 3. The project is licensed under the MIT license which allows the use of the code for educational and commercial purposes. 
\ No newline at end of file
+The purpose of the project is to provide clean and minimalistic implementations of basic algorithms and data structures in Python 3. The project is licensed under the MIT license which allows the use of the code for educational and commercial purposes. The project uses Sphinx to generate documentation, github actions for continuous delivery and flake8 for unit testing. The project is licensed under the permissive MIT license.
```

Comment:
The last sentence in the paragraph seems to be redundant.

Reply:
> Overall, the text is relevant and the figure gives a good overview of the project. I do, however, think the document needs to be longer in order to satisfy the P+ requirement (1-1,5 pages). It would also be easier to judge the length of the text if it were uploaded as a pdf.

I've extended the text!

Reply:
Your extensions to the text look good. I still think the last sentence in the first paragraph seems redundant since you already mention the MIT license earlier in the same paragraph. Other than that and the typo I mentioned in another comment, I think the branch is ready to be merged.

Reply:
Thank you. I've removed the redundant text and corrected the typo.